doc/Manuals/ug-cs_x-road_6_central_server_user_guide.md

```diff
@@ -430,7 +431,9 @@ To change the Central Server address, follow these steps.
   - generates new configuration anchors for the internal and external configuration sources.
 4. After the Central Server address is changed, act as follows.
   - Download the internal configuration source anchor and distribute the anchor along with the anchor’s hash value to the Security Server administrators of the local X-Road infrastructure.
+    > **NOTE**: Starting from version 7.5.0 new Central Server address is automatically distributed to Security Servers within the global configuration. Distribution will take place within two global configuration refresh cycles.
   - In case of federated X-Road systems, download the external configuration source anchor and distribute the anchor along with the anchor’s hash value to the federation partners.
+    > **NOTE**: Starting from version 7.5.0 new Central Server address is automatically distributed to the Configuration Proxy within the global configuration. Distribution will take place within two global configuration refresh cycles.
```

Comment:
I think it would be good to clarify the case when Configuration Proxy is not used in version 7.5.0 or later:

> **NOTE**: Starting from version 7.5.0 new Central Server address is automatically distributed to the Configuration Proxy within the global configuration. Distribution will take place within two global configuration refresh cycles. If Configuration Proxy is not used, distributing the anchor to the federation partners is required.